wdl/MatrixQC.wdl

```diff
@@ -147,16 +153,27 @@ task PESRBAF_QC {
   }
   RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
 
+  Float mem_gb = select_first([runtime_attr.mem_gb, default_attr.mem_gb])
+  Int java_mem_mb = ceil(mem_gb * 1000 * 0.8)
+
   output {
     File stats = "~{batch}.~{ev}.QC_matrix.txt"
   }
   command <<<
 
     set -euo pipefail
-    fgrep -v "#" ~{genome_file} | awk -v distance=~{distance} -v OFS="\t" '{ print $1, $2-distance, $2 }' > regions.bed;
-    GCS_OAUTH_TOKEN=`gcloud auth application-default print-access-token` \
-      tabix -R regions.bed ~{matrix_file} | bgzip -c -f > local_matrix.txt.gz
-    tabix -s 1 -b 2 -e 2 -f local_matrix.txt.gz;
+    fgrep -v "#" ~{genome_file} | awk -v distance=~{distance} -v OFS="\t" '{ print $1, $2-distance, $2 }' > regions.bed
+
+    java -Xmx~{java_mem_mb}M -jar ${GATK_JAR} LocalizeSVEvidence \
+      --sequence-dictionary ~{ref_dict} \
+      --evidence-file ~{matrix_file} \
+      -L regions.bed \
+      -O local_matrix.txt
+
+    # GATK does not block compress
+    bgzip local_matrix.txt
+    tabix -s 1 -b 2 -e 2 local_matrix.txt.gz
```

Comment:
Is this a bed file? Should it be `-e 3` (or `-p bed`)?

Reply:
Only bincov is bed type, the others do not have a proper END column